<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AModelPageClass</class>
 <widget class="QWidget" name="AModelPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Goal expression</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Model:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upAbstractModelPushButton">
          <property name="text">
           <string>&amp;Abstract</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upLinearModelPushButton">
          <property name="text">
           <string>&amp;Concrete &amp;Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upCubicModelPushButton">
          <property name="text">
           <string>Concrete &amp;Cubic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upCartLtAbstractModelPushButton">
          <property name="text">
           <string>CAR&amp;T lt Abstract</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="upGoalExprTextEdit">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Training options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Regression type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="upRegressionTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Dependent variable:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="upDependentVarComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Maximum generations:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="upMaxGenSpinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Halting fitness:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="upHaltingFitnessSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>upAbstractModelPushButton</tabstop>
  <tabstop>upLinearModelPushButton</tabstop>
  <tabstop>upCubicModelPushButton</tabstop>
  <tabstop>upCartLtAbstractModelPushButton</tabstop>
  <tabstop>upGoalExprTextEdit</tabstop>
  <tabstop>upRegressionTypeComboBox</tabstop>
  <tabstop>upDependentVarComboBox</tabstop>
  <tabstop>upMaxGenSpinBox</tabstop>
  <tabstop>upHaltingFitnessSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
